test/GitcoinGovernor.t.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import "forge-std/Test.sol";
+import {GitcoinGovernor} from "src/GitcoinGovernor.sol";
+
+contract GitcoinGovernorTest is Test {
+  uint256 VOTING_DELAY = 13_140;
+  uint256 VOTING_PERIOD = 40_320;
+  uint256 PROPOSAL_THRESHOLD = 1_000_000e18;
+  uint256 QUORUM = 2_500_000_000_000_000_000_000_000;
```

Comment:
```suggestion
  uint256 QUORUM = 2_500_000e18;
```

Reply:
Done in 78a11b444add9c6e0fb59e6751995cb55d6c82dd